lib/utils/templatize.html

```diff
@@ -96,15 +97,13 @@
        */
       _configureProperties(props) {
         let options = this.__templatizeOptions;
-        if (props) {
-          for (let iprop in options.instanceProps) {
-            if (iprop in props) {
-              this._setPendingProperty(iprop, props[iprop]);
-            }
+        if (options.forwardHostProp) {
+          for (let hprop in this.__hostProps) {
+            this._setPendingProperty(hprop, this.__dataHost['_host_' + hprop]);
           }
         }
-        for (let hprop in this.__hostProps) {
-          this._setPendingProperty(hprop, this.__dataHost['_host_' + hprop]);
+        for (let iprop in props) {
```

Comment:
Let's add a comment saying that passing in a prop in `props` that has the same name as a bound host prop may shadow it. This is considered a user error (warn?). We should add this to the docs too.